ansible/roles/schulcloud-server-core/templates/tldraw-deployment.yml.j2

```
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tldraw-deployment
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: tldraw-server
+    app.kubernetes.io/part-of: schulcloud-verbund
+    app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+    app.kubernetes.io/name: tldraw-server
+    app.kubernetes.io/component: tldraw
+    app.kubernetes.io/managed-by: ansible
+    git.branch: {{ SCHULCLOUD_SERVER_BRANCH_NAME }}
+    git.repo: {{ SCHULCLOUD_SERVER_REPO_NAME }}
+spec:
+  replicas: {{ TLDRAW_SERVER_REPLICAS|default("1", true) }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      #maxUnavailable: 1
+  revisionHistoryLimit: 4
+  paused: false
+  selector:
+    matchLabels:
+      app: tldraw-server
+  template:
+    metadata:
+      labels:
+        app: tldraw-server
```

Add labels to the pod
```suggestion
        app: tldraw-server
        app.kubernetes.io/part-of: schulcloud-verbund
        app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
        app.kubernetes.io/name: tldraw-server
        app.kubernetes.io/component: tldraw
        app.kubernetes.io/managed-by: ansible
        git.branch: {{ SCHULCLOUD_SERVER_BRANCH_NAME }}
        git.repo: {{ SCHULCLOUD_SERVER_REPO_NAME }}
```